packages/jest-cli/src/__tests__/watch-filename-pattern-mode-test.js

```diff
@@ -145,6 +145,8 @@ describe('Watch mode flows', () => {
   });
 
   it('can select a specific file name from the typeahead results', () => {
+    const toUnixPathPattern = pathPattern => pathPattern.replace(/\\\\/g, '/');
```

Comment:
Can you explain why we need to replace *two* backslashes with a single slash? Where does the second backslash come from?